origins_content_issue/src/Controller/ContentIssueCommentController.php

```diff
@@ -0,0 +1,158 @@
+   */
+  public function __invoke(): array {
+
+    $build['content'] = [
+      '#type' => 'item',
+      '#markup' => $this->t('It works!'),
+    ];
+
+    return $build;
+  }
+
+  /**
+   * Displays a create new comment form.
+   *
+   * @param int|string $issue_id
+   *   The id of the issue this comment is for.
+   */
+  public function addForm(int|string $issue_id) {
+    $issue_comment = $this->entityTypeManager()->getStorage('content_issue_comment')->create([]);
+
+    $form_state['values']['issue_entity'] = $issue_id;
+    $form = \Drupal::service('entity.form_builder')->getForm($issue_comment, 'add', $form_state);
+
+    $form['assigned_to']['#attributes']['class'][] = 'hidden';
+    $form["description"]["widget"][0]["format"]['#attributes']['class'][] = 'hidden';
+
+    return $form;
+  }
+
+  /**
+   * Displays an edit form for comments.
+   *
+   * @param int|string $comment_id
+   *   The id of the comment to edit.
+   */
+  public function editForm(int|string $comment_id) {
+    $comment = \Drupal::entityTypeManager()->getStorage('content_issue_comment')->load($comment_id);
+
+    // @phpstan-ignore-next-line
+    $form_state['values']['issue_entity'] = $comment->get('issue_entity')->getString();
+    $form_state['values']['comment_id'] = $comment_id;
+
+    $form = \Drupal::service('entity.form_builder')->getForm($comment, 'add', $form_state);
+
+    unset($form['actions']['delete']);
+
+    return $form;
+  }
+
+  /**
+   * Displays a delete confirmation form for comments.
+   *
+   * @param int|string $comment_id
+   *   The id of the comment to delete.
+   */
+  public function deleteConfirm(int|string $comment_id) {
+    $form = [
+      '#type' => 'container',
+      '#attributes' => ['class' => ['confirm-modal']],
+      'message' => [
+        '#markup' => '<p>Are you sure you want to proceed?</p>',
+      ],
+      'actions' => [
+        '#type' => 'container',
+        '#attributes' => ['class' => ['modal-actions']],
+        'confirm' => [
+          '#type' => 'link',
+          '#title' => $this->t('Confirm'),
+          '#url' => Url::fromRoute('origins_content_issue.comment.delete', ['comment_id' => $comment_id]),
+          '#attributes' => [
+            'class' => ['use-ajax', 'button', 'button--primary'],
+            'data-dialog-close' => 'true',
+          ],
+        ],
+        'cancel' => [
+          '#type' => 'link',
+          '#title' => $this->t('Cancel'),
+          '#url' => Url::fromRoute('origins_content_issue.comment.close_modal'),
+          '#attributes' => [
+            'class' => ['use-ajax'],
+            'data-dialog-close' => 'true',
+          ],
+        ],
+      ],
+    ];
+
+    $form['comment_id'] = [
+      '#type' => 'hidden',
+      '#value' => $comment_id,
+    ];
+
+    return $form;
+  }
+
+  /**
+   * Deletes a comment and closes the modal.
+   *
+   * @param int|string $comment_id
+   *   The id of the comment to delete.
+   */
+  public function delete($comment_id) {
```

Comment:
add the parameter type in the function signature as well as the annotation? 

`public function delete(int|string $comment_id)`